mobile/lib/features/text_management/presentation/bloc/text_state.dart

```diff
@@ -22,6 +16,9 @@ class TextCreated extends TextState {
   final MyText text;
 
   TextCreated({@required this.text});
+
+  @override
+  List<Object> get props => [text];
 }
 
 class UpdatingText extends TextState {}
```

Comment:
Why does the `Updating`, `Getting` states holds values? Shouldn't only the `Updated` and `Got` states return the new values?